hack/patches/009-workloads-probes.patch

```diff
@@ -322,6 +322,44 @@ index 257c3757f..605777742 100644
 +	in.DeepCopyInto(out)
 +	return out
 +}
+diff --git a/vendor/knative.dev/operator/pkg/apis/operator/v1alpha1/deploymentoverride.go b/vendor/knative.dev/operator/pkg/apis/operator/v1alpha1/deploymentoverride.go
+index 18f5d27b7..f732fa1c2 100644
+--- a/vendor/knative.dev/operator/pkg/apis/operator/v1alpha1/deploymentoverride.go
++++ b/vendor/knative.dev/operator/pkg/apis/operator/v1alpha1/deploymentoverride.go
+@@ -21,10 +21,10 @@ import (
+ )
+
+ // ConvertToDeploymentOverride merges the ResourceRequirementsOverride into the DeploymentOverride
+-func ConvertToDeploymentOverride(source base.KComponent) []base.DeploymentOverride {
+-	mergedDeploymentOverride := source.GetSpec().GetDeploymentOverride()
++func ConvertToDeploymentOverride(source base.KComponent) []base.WorkloadOverride {
```

Comment:
wasn't the workload override already done in upstream?

Reply:
In 1.5.2 no?